javascript/linkedList/linkedList/README.md

```diff
@@ -70,4 +70,16 @@ Write a method for the Linked List class which takes a number, k, as a parameter
 My partner and I whiteboarded for an hour, and came up with the solution below. We thought we should traverse the LL to calculate its length, then subtract K from that length, then traverse the LL to that position. Since we will be traversing the LL twice, the Big O is 0(n^2).
 
 ## Solution
-![LL k-th value whiteboard](./assets/ll-kth-from-end.png)
\ No newline at end of file
+![LL k-th value whiteboard](./assets/ll-kth-from-end.png)
+
+# Challenge Summary
+Zip two linked lists.
+
+## Challenge Description
+Write a function called zipLists which takes two linked lists as arguments. Zip the two linked lists together into one so that the nodes alternate between the two lists and return a reference to the head of the zipped list. Try and keep additional space down to O(1). You have access to the Node class and all the properties on the Linked List class as well as the methods created in previous challenges.
+
+## Approach & Efficiency
+My partner and I whiteboarded for an hour, and didn't get as far ad we'd hoped to. We were contemplatng how to integrate some of the other methods into our solution and ran out of time. 
+
+## Solution
+![LL k-th value whiteboard](./assets/LL-zip.png)
```

Comment:
Really great looking whiteboard! I would suggest changing the alt text to the LL zip or something similar!
Would like to see code and tests added!